app/cafe.py

```diff
@@ -0,0 +1,19 @@
+from app.errors import NotVaccinatedError, \
+                        OutdatedVaccineError, \
+                        NotWearingMaskError
+import datetime
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, vstr: dict) -> str:
+        if "vaccine" not in vstr.keys() or vstr["vaccine"] == None:
```

Comment:
It is recommended to use `is None` instead of `== None`. Check PEP8 for details:
https://peps.python.org/pep-0008/#programming-recommendations

Besides this, if you create a class like this
```
class UniversallyEqual():
    def __eq__(self, other):
        return True
```
its instances will be equal to any object including `None` but they will not be None: try `UniversallyEqual() == None` and `UniversallyEqual() is None` to see the difference.

You can also use `vstr.get("vaccine")` - it can simplify the condition :)